libnavui-androidauto/build.gradle

```diff
@@ -73,7 +73,7 @@ dokkaHtmlPartial {
         configureEach {
             reportUndocumented.set(true)
             perPackageOption {
-                matchingRegex.set("com.mapbox.navigation.ui.car.internal.*")
+                matchingRegex.set("com.mapbox.androidauto.internal.*")
```

Comment:
At the moment this isn't needed as there are no "public" APIs under the `internal` package. See my comment above.

Reply:
Yeah, am only adding this so that there is a place to put "internal" apis that are still public. I considered moving all apis to this package at first. But decided to evaluate each public api to prevent breaking 1tap

Reply:
I moved the logger to internal for now. That way this package has a purpose while we make more progress